### PR TITLE
feat: add db-brand, db-header, db-page

### PR DESCRIPTION
**Pull request by nmerget**

### CODE REVIEW

File: packages/components/src/shared/model.ts [hunk 1]
```diff
@@ -15,6 +15,13 @@ export type GlobalTextProps = {
 	pattern?: string;
 };
 
+export type ImageProps = {
+	imgAlt?: string;
+	imgSrc?: string;
+	imgHeight?: number | string;
+	imgWidth?: number | string;
```
Comment (mfranzke) [inline packages/components/src/shared/model.ts:22]:
```suggestion
	imgHeight?: number;
	imgWidth?: number;
```

> The intrinsic width of the image in pixels. Must be an integer without a unit.

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/img#attr-width

**Review by mfranzke — DISMISSED** [review]

File: packages/components/src/shared/model.ts (hunk 1 above)
Comment (mfranzke) [reply packages/components/src/shared/model.ts:22]: how is this resolved ?

**Review by mfranzke — APPROVED** [review]

File: showcases/react-showcase/src/app.tsx [hunk 2]
```diff
@@ -12,97 +15,116 @@ const App = () => {
...
-				</section>
+						<DBInput
+							label="Startdatum"
+							placeholder="irgendein Text"
+							type="datetime-local"
+							id="input-func-date"
+						/>
+					</section>
+				</div>
 			</div>
```
Comment (mfranzke) [inline showcases/react-showcase/src/app.tsx:126]:
```suggestion
			</main>
```

every page should contain a `<main>` element, mainly for a11y purpose to act as a landmark.

**Review by mfranzke — CHANGES_REQUESTED** [review]

File: showcases/react-showcase/src/app.tsx (hunk 2 above)
Comment (nmerget) [reply showcases/react-showcase/src/app.tsx:126]: Had to revert it...
`<DBPage>` has a `<main>` tag already: https://github.com/db-ui/mono/blob/feat-db-brand/packages/components/src/components/page/page.lite.tsx#L40

We gonna refactor the showcases in the future. At the part you mentioned would be a native router outlet (inside the main tag) that loads another component (content-site) based on the route. So I think everything is fine here

**Review by mfranzke — APPROVED** [review]

**Pull request merged**